cordformation/src/main/kotlin/net/corda/plugins/DockerImage.kt

```diff
@@ -0,0 +1,100 @@
+package net.corda.plugins
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+open class DockerImage : DefaultTask() {
+
+    private companion object{
+        private const val DOCKER_FILE_NAME = "dockerfile"
+        private const val CORDA_VERSION_PROP = "corda_release_version"
+    }
+
+    private enum class Images(val label: String) {
+        ZULU("corda/corda-zulu-java1.8"),
```

Comment:
I see hard-coded strings containing values like "zulu" and "java1.8". And I am afraid... _very afraid_.... Aren't these image names likely to change?

Reply:
Removed